src/olympia/signing/tests/test_views.py

```diff
@@ -453,7 +453,7 @@ def test_version_exists_with_pk(self):
         response = self.get(self.url(self.guid, '3.0', upload.uuid))
         assert response.status_code == 200
         # For backwards-compatibility reasons, we return the uuid as "pk".
-        assert response.data['pk'] == upload.uuid
+        assert response.data['pk'] == str(upload.uuid)
```

Comment:
gosh, good call - I was under the impression the signing api returned with the dashes - checked docs and that's not the case. I'll fix this.